dataland-frontend/tests/e2e/specs/framework-view-and-upload-journey/UploadFrameworkDataJourney.ts

```diff
@@ -73,27 +75,22 @@ describe('As a user, I expect the dataset upload process to behave as I expect',
               return cy
                 .wait(timeDelayInMillisecondsBeforeNextUploadToAssureDifferentTimestamps)
                 .then(() => {
-                  const preparedFixture = getPreparedFixture(
-                    'eligible-activity-Point-26',
-                    euTaxoFinancialPreparedFixtures
-                  );
-                  return uploadFrameworkDataForLegacyFramework(
-                    DataTypeEnum.EutaxonomyFinancials,
+                  return uploadFrameworkDataForPublicToolboxFramework(
+                    EuTaxonomyFinancialsBaseFrameworkDefinition,
                     token,
                     storedCompanyForManyDatasetsCompany.companyId,
                     '2022',
-                    preparedFixture.t
+                    euTaxoFinancialPreparedFixture.t
                   );
                 })
                 .then((dataMetaInformationOfSecondUpload) => {
                   dataIdOfSecondEuTaxoFinancialsUpload = dataMetaInformationOfSecondUpload.dataId;
-                  const preparedFixture = getPreparedFixture('LkSG-date-2022-07-30', lksgPreparedFixtures);
                   return uploadFrameworkDataForPublicToolboxFramework(
                     LksgBaseFrameworkDefinition,
                     token,
                     storedCompanyForManyDatasetsCompany.companyId,
                     generateReportingPeriod(),
-                    preparedFixture.t
+                    lksgPreparedFixture.t
```

Comment:
Wurden in dem Test vorher EU-Taxonomie-Daten als LkSG-Daten hochgeladen?

Reply:
Nein es wurden auch vorher schon lksg Daten hochgeladen.

`preparedFixture` war eine lokale `const`, in der ein lksg Datensatz steckte.